test/common/upstream/maglev_lb_test.cc

```diff
@@ -63,6 +66,12 @@ TEST_F(MaglevLoadBalancerTest, NoHost) {
   EXPECT_EQ(nullptr, lb_->factory()->create()->chooseHost(nullptr));
 };
 
+// Throws an exception if table size is not a prime number.
+TEST_F(MaglevLoadBalancerTest, NoPrimeNumber) {
+  EXPECT_THROW_WITH_MESSAGE(init(8), EnvoyException,
```

Comment:
Even if the new config is added, so the table size of Maglev algorithm is no changeable, no need to add new tests for that because existing tests are already testing the Maglev algorithm with many table size values.